Source/com/drew/metadata/avi/AviDirectory.java

```diff
@@ -0,0 +1,51 @@
+package com.drew.metadata.avi;
+
+import com.drew.lang.annotations.NotNull;
+import com.drew.metadata.Directory;
+
+import java.util.HashMap;
+
+/**
+ * @author Payton Garland
+ */
+public class AviDirectory extends Directory
+{
+
+    public static final int TAG_FRAMES_PER_SECOND = 1;
+    public static final int TAG_SAMPLES_PER_SECOND = 2;
+    public static final int TAG_DURATION = 3;
+    public static final int TAG_VIDEO_CODEC = 4;
+    public static final int TAG_AUDIO_CODEC = 5;
+    public static final int TAG_WIDTH_PIX = 6;
+    public static final int TAG_HEIGHT_PIX = 7;
+    public static final int TAG_STREAMS = 8;
+
+    @NotNull
+    protected static final HashMap<Integer, String> _tagNameMap = new HashMap<Integer, String>();
+
+    static {
+        _tagNameMap.put(TAG_FRAMES_PER_SECOND, "Frames Per Second");
+        _tagNameMap.put(TAG_SAMPLES_PER_SECOND, "Samples Per Second");
+        _tagNameMap.put(TAG_DURATION, "Duration");
+        _tagNameMap.put(TAG_VIDEO_CODEC, "Video Codec");
+        _tagNameMap.put(TAG_AUDIO_CODEC, "Audio Codec");
+        _tagNameMap.put(TAG_WIDTH_PIX, "Width (pixels)");
```

Comment:
Minor point: In general tag names don't include units (such as pixels). Units are included in the description text.